src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.Unix.cs

```diff
@@ -405,7 +405,7 @@ public static void MoveDirectory(string sourceFullPath, string destFullPath)
                     case Interop.Error.EACCES: // match Win32 exception
                         throw new IOException(SR.Format(SR.UnauthorizedAccess_IODenied_Path, sourceFullPath), errorInfo.RawErrno);
                     default:
-                        throw Interop.GetExceptionForIoErrno(errorInfo, sourceFullPath, isDirectory: true);
+                        throw Interop.GetExceptionForIoErrno(errorInfo, isDirectory: true);
```

Comment:
@tmds, you ok with this?  Seems reasonable, if for no other reason than there are two paths involved and we're sort of arbitrarily picking one to include in the error.

Reply:
Sure. It makes sense to include both or neither.

Reply:
I made it to where there were zero paths included to match what is on the Windows side. Is this able to merge now?